dotcom-rendering/src/web/components/Section.tsx

```diff
@@ -14,33 +14,90 @@ import { ShowHideButton } from './ShowHideButton';
 import { Treats } from './Treats';
 
```

Comment:
Could you add a comment here something like

```tsx
/**
 * ----- First time here? -----
 * If you're unsure how to use this component, check out `Section.stories.tsx` and
 * try running it in storybook to get an idea of how this component is used!
 */
```

I am conscious this component is very large and unwieldy - and giving future developers a small hint on how to start working with it might be quite helpful!

Reply:
Sounds like a good idea.

I'm also thinking it might be nice - at a later stage - to break things up into `FrontSection` (or maybe `Container`) and `Section` so that all the front specific logic can live by itself. I haven't really got my head into that though